### PR TITLE
Particle pos arguments

### PR DESCRIPTION
**Pull request by YanisBft**
Also renamed TurtleEntityModel to be consistent with other model classes
Mapped missing block sound group

### CODE REVIEW

**Review by Runemoro — APPROVED** [review]

File: mappings/net/minecraft/world/LightType.mapping [hunk 1]
```diff
@@ -1 +1,4 @@
 CLASS bhr net/minecraft/world/LightType
+	FIELD c LIGHT I
```
Comment (Gegy) [inline mappings/net/minecraft/world/LightType.mapping]: What's this constant? Maybe missing context but `LIGHT` seems pretty vague

Comment (YanisBft) [reply mappings/net/minecraft/world/LightType.mapping]: Luminance better?

Comment (Gegy) [reply mappings/net/minecraft/world/LightType.mapping]: How is it used?

Comment (YanisBft) [reply mappings/net/minecraft/world/LightType.mapping]: It seems to be just used in LightType class, in the constructor

Comment (Gegy) [reply mappings/net/minecraft/world/LightType.mapping]: It's not a `static final` constant, it should be camelCase. The value doesn't seem to be used anywhere (unless Enigma is reporting wrong) I think it's best to not map it meaninglessly.

**Review by Gegy — APPROVED** [review]

**Pull request merged**